app/models/user.rb

```diff
@@ -6,16 +6,22 @@ class User < ApplicationRecord
   VALIDATE_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
 
   has_secure_password
+
   before_save :downcase
 
   validates :name, presence: true, length: { maximum: Settings.DIGIT_50 }
 
   validates :email, presence: true, length: { maximum: Settings.DIGIT_255 },
-                    format: { with: VALIDATE_EMAIL_REGEX }, uniqueness: true
+            format: { with: VALIDATE_EMAIL_REGEX }, uniqueness: true
+  validates :password, presence: true,
+            length: { minimum: Settings.DIGIT_6 }, allow_nil: true
 
   validates :birthday, :gender, presence: true
+
   validate :birthday_within_last_100_years
```

Comment:
bỏ check ở line 56 chỗ present? em đưa lên đây dc 
```suggestion
  validate :birthday_within_last_100_years, if: -> {birthday.present?}
```